delivery_translatable/README.rst

```diff
@@ -0,0 +1,5 @@
+=====================
+Delivery Translatable
+=====================
+
+* name and website_description field of delivery carrier setup translatable.
```

Comment:
This has to be moved (without the title) to the `readme/` subfolder, in `readme/DESCRIPTION.rst` (check other modules in this repository how it is done). Then the `README.rst` will be generated automatically by the OCA bot.